src/status_im/contexts/profile/contact/contact_request/view.cljs

```diff
@@ -40,20 +50,24 @@
       (i18n/label :t/contact-request-message-prompt)]
      [rn/view {:style style/message-input-wrapper}
       [quo/input
-       {:type                :text
-        :multiline?          true
-        :char-limit          constants/contact-request-message-max-length
-        :max-length          constants/contact-request-message-max-length
-        :placeholder         (i18n/label :t/type-something)
-        :auto-focus          true
-        :accessibility-label :contact-request-message
-        :label               (i18n/label :t/message)
-        :on-change-text      on-message-change}]]
-     [quo/bottom-actions
-      {:actions          :one-action
-       :button-one-props {:disabled?           (string/blank? message)
-                          :accessibility-label :send-contact-request
-                          :customization-color customization-color
-                          :on-press            on-message-submit}
-       :button-one-label (i18n/label :t/send-contact-request)}]]))
+       {:type                  :text
+        :ref                   #(reset! input-ref %)
+        :multiline?            true
+        :char-limit            constants/contact-request-message-max-length
+        :max-length            constants/contact-request-message-max-length
+        :placeholder           (i18n/label :t/type-something)
+        :auto-focus            true
+        :accessibility-label   :contact-request-message
+        :label                 (i18n/label :t/message)
+        :on-change-text        on-message-change
+        :container-style       {:flex-shrink 1}
+        :input-container-style {:flex-shrink 1}}]]
+     [rn/view {:style {:flex 1}}
+      [quo/bottom-actions
```

Comment:
bottom actions doesn't have a container-style instead of the wrapping rn/view?? 🤔